rfcs/packaging/communicating-convergence-status.md

```diff
@@ -0,0 +1,141 @@
+# RFC: Communicating convergence status for `@fluentui/*` packages (and related code moves)
+
+**Note that this RFC focuses solely on near-term steps to clearly communicate to consumers which of our packages are old vs. converged. It's focused primarily on v7/8 and converged packages and does not deal with folder structure or northstar packages.**
+
+**[Related RFC around other package structure issues during convergence](https://github.com/microsoft/fluentui/pull/16577)**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+Currently, we have a lot of different component packages, and their purpose, production-readiness, and convergence status are not clear to consumers or contributors.
+
+There are a few aspects to the proposed solution, involving a mix of code moves and documentation updates:
+
+- "Converged" component packages should only contain convergence work
+  - _(done)_ ~~For `react-{checkbox,link,slider,tabs,toggle}`:~~
+    - ~~Move old (v8) component implementations back to `react-internal`~~
+    - ~~Move prototype "converged" versions to the package roots and get rid of `/next` folders~~
+  - Get rid of `react-internal`
+    - `react-internal` components move back to `react`
+    - To break a cycle: `react-date-time` components move into `react`, and `react-date-time` re-exports them
+  - Open question: should we make code location or naming changes to any other `react-*` packages which still contain legacy components?
```

Comment:
I would say it would be ideal if the only `react-*` packages were converged components, but I'll leave it to you whether this is feasible to do within the time we have before v8 release. I don't think we should push out v8 release for moving other `react-*` packages.

Reply:
Keep this because we will have a variety of other platforms (inside and outside our repo) under `@fluentui` scope.

Would be nice if it could mean "converged" but that's not realistic.